src/status_im2/contexts/share/view.cljs

```diff
@@ -0,0 +1,163 @@
+(ns status-im2.contexts.share.view
+  (:require [utils.i18n :as i18n]
+            [quo2.core :as quo]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im2.contexts.share.style :as style]
+            [utils.re-frame :as rf]
+            [reagent.core :as reagent]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.qr-code.qr :as qr]
+            [clojure.string :as string]
+            ;;TODO(siddarthkay) : move the components below over to status-im2 ns
+            ;; issue -> https://github.com/status-im/status-mobile/issues/15549
+            [status-im.ui.components.list-selection :as list-selection]
+            [status-im.ui.components.react :as react]))
+
+(def ^:const profile-tab-id 0)
+(def ^:const wallet-tab-id 1)
+
+(defn copy-text-and-show-toast
+  [{:keys [text-to-copy post-copy-message]}]
+  (react/copy-to-clipboard text-to-copy)
+  (rf/dispatch [:share/show-successfully-copied-toast post-copy-message]))
+
+(defn header
+  []
+  [rn/view
+   [quo/button
+    {:icon                true
+     :type                :blur-bg
+     :size                32
+     :accessibility-label :close-activity-center
+     :override-theme      :dark
+     :style               style/header-button
+     :on-press            #(rf/dispatch [:hide-popover])}
+    :i/close]
+   [quo/text
+    {:size   :heading-1
+     :weight :semi-bold
+     :style  style/header-heading}
+    (i18n/label :t/share)]])
+
+(defn profile-tab
+  [window-width]
+  (let [multiaccount              (rf/sub [:multiaccount])
+        emoji-hash                (string/join "" (get multiaccount :emoji-hash))
```

Comment:
`string/join` defaults to joining collections without spaces, so you can safely remove `""`.

Reply:
resolved here: https://github.com/status-im/status-mobile/pull/15528/commits/bcd8198f1ece5a75ad440c54b4e2f07ff372da1f